go-controller/pkg/metrics/metrics.go

```diff
@@ -166,6 +166,16 @@ func checkPodRunsOnGivenNode(clientset kubernetes.Interface, label, k8sNodeName
 	return false, fmt.Errorf("the Pod matching the label %q doesn't exist on this node %s", label, k8sNodeName)
 }
 
+var ovnRegistry = prometheus.NewRegistry()
+
+func MakeOvnRegistryPointToDefault() {
```

Comment:
This is not used anymore, you can remove it

Reply:
Done.